api/sixtyfps-cpp/tests/datastructures.cpp

```diff
@@ -13,6 +13,7 @@ LICENSE END */
 #include "catch2/catch.hpp"
 
 #include <sixtyfps.h>
+#include <sixtyfps_image.h>
```

Comment:
Strange that this is needed, i'd expect sixtyfps.h to be sufficient

Reply:
The same applies to brush I think. We can change that though. I’d do that separate from this merge request though.